crates/turborepo-lib/src/shim.rs

```diff
@@ -407,11 +409,35 @@ impl RepoState {
                 self.spawn_local_turbo(&canonical_local_turbo, shim_args),
             ))
         } else {
-            try_check_for_updates(&shim_args, get_version());
+            let global_version = get_version();
```

Comment:
I feel like this should be a proc-macro. We should be confirming that it is valid at compile time.

But that's not for this PR.